test/MobxPromise.spec.ts

```diff
@@ -96,4 +96,19 @@ describe('MobxPromise', () => {
 			}, 200);
 		});
 	});
+
+	it('calls invoke with the arguments from await', (done)=>{
+		let mp1 = new MobxPromise(()=>new Promise<number>(resolve=>setTimeout(()=>resolve(5), 10)));
+		let mp2 = new MobxPromise(()=>new Promise<string>(resolve=>setTimeout(()=>resolve("hello"), 10)));
+		let mp3 = new MobxPromise({
+			await:()=>[mp1, mp2],
+			invoke:(a0:number, a1:string)=>{
```

Comment:
confirmed that typescript is doing good inference here - if i change one of the argument types, TS fails; if i add an argument, TS fails